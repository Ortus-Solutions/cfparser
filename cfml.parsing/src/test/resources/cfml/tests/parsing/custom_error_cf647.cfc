  try {
  } catch (org.example.whateverError) {
  } catch (org.example.whateverError local.foo) {
  }