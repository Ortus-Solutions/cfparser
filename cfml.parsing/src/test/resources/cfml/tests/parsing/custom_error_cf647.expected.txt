/*===TOKENS===*/
TRY                      <try>
'{'                      <{>
Hidden:NEWLINE                  <>
'}'                      <}>
CATCH                    <catch>
'('                      <(>
IDENTIFIER               <org>
'.'                      <.>
IDENTIFIER               <example>
'.'                      <.>
IDENTIFIER               <whateverError>
')'                      <)>
'{'                      <{>
Hidden:NEWLINE                  <>
'}'                      <}>
CATCH                    <catch>
'('                      <(>
IDENTIFIER               <org>
'.'                      <.>
IDENTIFIER               <example>
'.'                      <.>
IDENTIFIER               <whateverError>
IDENTIFIER               <local>
'.'                      <.>
IDENTIFIER               <foo>
')'                      <)>
'{'                      <{>
Hidden:NEWLINE                  <>
'}'                      <}>
/*===TREE===*/
(scriptBlock
  (element
    (statement
      (tryCatchStatement
        try
        (statement (compoundStatement { }))
        (catchCondition
          catch
          (
          (multipartIdentifier
            (identifier org)
            .
            (identifierOrReservedWord (identifier example))
            .
            (identifierOrReservedWord (identifier whateverError))
          )
          )
          (compoundStatement { })
        )
        (catchCondition
          catch
          (
          (typeSpec
            (multipartIdentifier
              (identifier org)
              .
              (identifierOrReservedWord (identifier example))
              .
              (identifierOrReservedWord (identifier whateverError))
            )
          )
          (multipartIdentifier
            (identifier local)
            .
            (identifierOrReservedWord (identifier foo))
          )
          )
          (compoundStatement { })
        )
      )
    )
  )
)
/*======*/
/*===DECOMPILE===*/
try{

}catch(org.example.whateverError)
{

}catch(org.example.whateverError local.foo)
{

}
/*======*/